airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/cdc/PostgresCdcProperties.java

```diff
@@ -68,6 +68,11 @@ private static Properties commonProperties(final JdbcDatabase database) {
             : HEARTBEAT_INTERVAL;
     props.setProperty("heartbeat.interval.ms", Long.toString(heartbeatInterval.toMillis()));
 
+    if (sourceConfig.get("replication_method").has("heartbeat_action_query")
```

Comment:
Can we add `heartbeat_action_query` to a test - even by just configuring it on a test to make sure nothing is broken.